app/views/user_mentee_applications/show.html.erb

```diff
@@ -1,7 +1,4 @@
 <div class="mx-auto p-4 my-8 max-w-prose">
-  <% if policy(:UserOnly).admin? %>
```

Comment:
A lot of the red here is a nice consequence of this separation. Admin users are guaranteed to be viewing the admin stuff, and applicant viewers are guaranteed to be viewing the applicant stuff -- all enforced at the controller level. Keeps these conditional checks out of the views :muscle: